### ISSUE TITLE
Could not find a declaration file for module 'tunnel'.

### ISSUE THREAD

**Issue by kpajdzik** [Azure/ms-rest-js#324]
**Package Version**: 1.5.1
**Describe the bug**
ms-rest-js@1.5.1 is missing type definitions for "tunnel" package. 

Example error:
```
> @azure/keyvault@0.1.0 build /home/vsts/work/1/s/packages/@azure/keyvault
> tsc && rollup -c rollup.config.js && npm run minify

node_modules/@azure/ms-rest-js/es/lib/axiosHttpClient.d.ts(6,25): error TS7016: Could not find a declaration file for module 'tunnel'. '/home/vsts/work/1/s/packages/@azure/keyvault/node_modules/tunnel/index.js' implicitly has an 'any' type.
  Try `npm install @types/tunnel` if it exists or add a new declaration (.d.ts) file containing `declare module 'tunnel';`
```

Fix is coming in 1.5.2

**Comment by kpajdzik** [comment]
Package was unpublished by Mike Harder.

**Comment by kpajdzik** [comment]
Fixed in 1.5.3
